src/pages/lantern/Lantern.tsx

```
@@ -82,13 +109,39 @@ const Lantern = () => {
 
     // 컴포넌트 마운트 후 즉시 실행
     updateCloseButtonRect();
-
     window.addEventListener('resize', updateCloseButtonRect);
+
+      lanternsList.push(  { lantern_id: currentLanternId as string, owner_name: (currentLanternId as string).slice(0, -5), emotion: '', is_current_lantern: false });
```

ESLint

[error] Multiple spaces found before '{'.

(no-multi-spaces)